source/actors/Gravity.gd

```diff
@@ -0,0 +1,10 @@
+extends "res://actors/PlatformPhysics.gd"
+
+export (float) var strength = 1200.0
+export (Vector2) var direction = Vector2.DOWN
+
+func _physics_process(delta):
+	if kinematic_body2d.is_on_floor():
+		kinematic_body2d.velocity.y = 0
```

Comment:
Maybe adjust this in the `PlatformActor.gd` to make it update its `velocity` to be the returning `Vector2` of the `move_and_slide` method?